### ISSUE TITLE
Incorrect documentation ..  or sign of bug if docs are correct

### ISSUE THREAD

**Issue by louisconcentricsky** [jina-ai/docarray#268]
https://github.com/jina-ai/docarray/blob/510dfc934c63c9480a20e8003243c8abee12a675/docs/fundamentals/dataclass/index.md?plain=1#L175

The `tags` from the example object are put into a sibling document to the other fields in the chunks and are not associated with the container document. In fact the container document is pretty empty except for the chunks.

Not sure if the docs are wrong or they indicate the intended behavior and this is just a bug.

**Comment by hanxiao** [comment]
you are right, the documentation is wrong.

> In fact the container document is pretty empty except for the chunks.

btw, this is because `_` underscore attributes are not plotted in `.summary()`. otherwise it will be too much. So it looks empty as our prettify is designed as so.

**Comment by louisconcentricsky** [comment]
Pretty minor in the grand scheme of things, but thought I'd mention it as I noticed it right away. Intuitively my first instinct was that the fields stuffed into tags should be clustered on the parent doc as you think they'd apply to all the chunks and not be their "equivalent", but I have not spent any brain cycles on the design implications nor the intent so not married to that instinct.

**Comment by hanxiao** [comment]
oh actually.

**Comment by hanxiao** [comment]
you are wrong, it is not a bug or wrong documentation. 😆

**Comment by louisconcentricsky** [comment]
What I read is that the intent and my instinct match up, but the subtlety is that in the example the `meta` is a `JSON` type so created as a sibling in the `chunks`. Fair enough the API docs do indicate that. I see changing the demo to:

```
from docarray import dataclass, Document, DocumentArray
from docarray.typing import Image, Text


@dataclass
class WPArticle:
    banner: Image
    headline: Text
    author: str
    column: str


a = WPArticle(
    banner='/Volumes/Work/docarray-test/dog-cat-filght.png',
    headline='Everything to know about flying with pets, from picking your seats to keeping your animal calm',
    author= 'Nathan Diller',
    column= 'By the Way - A Post Travel Destination'
)

Document(a)
```

Get's me the other paradigm.

Might be helpful to call out both avenues of structuring data or use the above in the overview as it matches that design description.

**Comment by hanxiao** [comment]
oh this is because your `author` and `column` are Python primitive types, so they behave differently comparing to `JSON`

<img width="850" alt="image" src="https://user-images.githubusercontent.com/2041322/162289626-20d58e6e-f86f-4bae-980a-5b3891c1e254.png">

**Comment by louisconcentricsky** [comment]
I don't think we are communicating well. I understand how it all works and agree the examples all conform to the API doc. I still hold the front page code example doesn't well represent what is described in the paragraph

>  Having those in mind, to represent a multimodal document it seems that we need to put each modality as a separated Document and then nested them under a parent Document. For example, the article card from The Washington Post would be represented as follows:

The above code snippet I pasted would generate an example that does match. Figured it out by reading the API docs you linked.

In the end fell free to close this and move on as you prefer.

I do thank you for the time and attention you have given this little ticket and exercise. I did walk away understanding better how it works, and my suggestions are only put forth to make help the next person.
